### PR TITLE
add docstrings for functions get_particle_enery and calculate_total_p…

### PR DESCRIPTION
**Pull request by jRicciL**
Documentation added for functions calculate_total_pair_energy and get_particle_energy.

- [ ] Check spelling mistakes

### CODE REVIEW

File: day_1.py [hunk 1]
```diff
@@ -45,8 +45,26 @@ def minimum_image_distance(r_i, r_j, box_length):
...
-    #This function computes the energy of a particle with
-    #the rest of the system
+    ''' Computes the energy of a particle with the rest of the system.
+
+    Parameters
+    ----------
+    coordinates : np.array
+        An array of atomic coordinates (x, y, z). Shape (n, 3), where
+        n is the number of particles.
+    box_lenght : float
```
Comment (bdnguye2) [inline day_1.py]: typo to length

File: day_1.py [hunk 2]
```diff
@@ -69,6 +87,25 @@ def get_particle_energy(coordinates, box_length, i_particle, cutoff2):
...
+        Size of the box in Angstroms.
+    cutoff : float
+        Squared cutoff to evaluate Lennard Jones interaction between two particles.
+
+    Returns
+    -------
+    e_total : float
+        Total energy of particle i with the rest of the system.
```
Comment (bdnguye2) [inline day_1.py]: Total pair energy(?) not the total energy of any given particle

File: day_1.py [hunk 3]
```diff
@@ -69,6 +88,25 @@ def get_particle_energy(coordinates, box_length, i_particle, cutoff2):
...
 
 def calculate_total_pair_energy(coordinates, box_length, cutoff2):
+    ''' Computes the total energy between all pairs of molecules of whole system.
+    
+    ----------
+    Parameters
+    ----------
+    coordinates : np.array
+        An array of atomic coordinates (x, y, z). Shape (n, 3), where n is the number of particles.
+    box_lenght : float
```
Comment (b-butler) [inline day_1.py]: box length is misspelled and the length isn't in Angstrom.

File: day_1.py [hunk 4]
```diff
@@ -69,6 +88,25 @@ def get_particle_energy(coordinates, box_length, i_particle, cutoff2):
...
+        Size of the box in Angstroms.
+    cutoff : float
```
Comment (b-butler) [inline day_1.py]: cutoff2 not cutoff

File: day_1.py [hunk 5]
```diff
@@ -96,8 +96,27 @@ def minimum_image_distance(r_i, r_j, box_length):
...
+    ''' Computes the energy of a particle with the rest of the system.
+
+    ----------
+    Parameters
+    ----------
+    coordinates : np.array
+        An array of atomic coordinates (x, y, z). Shape (n, 3), where
+        n is the number of particles.
+    box_length : float
+        Size of the box in Angstroms.
```
Comment (b-butler) [inline day_1.py]: Not in Angstrom.

**Pull request merged**